app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
+  def help
+  end
```

Comment:
Đối với method rỗng em viết thành một dòng nhé, method cách nhau một dòng trống 

```suggestion
  def home; end
  
  def help; end
```

Reply:
oke anh :>